### ISSUE TITLE
Issues with recent builds

### ISSUE THREAD

**Issue by mjpitz** [depscloud/depscloud#103]
Opening for transparency. I've noticed some sporadic issues with our builds due to the fetch for go-groups.

Upstream issue: https://github.com/indeedeng/go-groups/issues/20

Some thoughts on how we can add stability:
- Build intermediary containers for our CI pipelines where possible
  - Should be easy to do with existing `dockerfiles` infrastructure
  - This will reduce our dependency on fetching from Indeed all the time

**Comment by abitrolly** [comment]
Is that still an issue?

**Comment by abitrolly** [comment]
So you mean https://oss.indeed.com/ is not a `depscloud` installation, but some kind of `go` server. Why it is not possible to `go build` directly from https://github.com/indeedeng/go-groups ?

**Comment by mjpitz** [comment]
Yes. And I can build directly from the repo, just requires a bit more work to do.
